sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -97,110 +93,86 @@ def __init__(self, client, source, **kwargs):
         self._owner_level = owner_level
         self._keep_alive = keep_alive
         self._auto_reconnect = auto_reconnect
-        self._retry_policy = errors.ErrorPolicy(
-            max_retries=self._client._config.max_retries,
-            on_error=_error_handler,  # pylint:disable=protected-access
-        )
+        self._retry_policy = self._amqp_transport.create_retry_policy(self._client._config)
         self._reconnect_backoff = 1
-        self._link_properties = {}  # type: Dict[types.AMQPType, types.AMQPType]
+        link_properties: Dict[uamqp_types.AMQPType, uamqp_types.AMQPType] = {}
         self._error = None
         self._timeout = 0
-        self._idle_timeout = (idle_timeout * 1000) if idle_timeout else None
+        self._idle_timeout = (idle_timeout * self._amqp_transport.IDLE_TIMEOUT_FACTOR) if idle_timeout else None
         partition = self._source.split("/")[-1]
         self._partition = partition
```

Comment:
Not related to your changes, is there a reason to have 102 and 103 as separate lines?

Reply:
🤔 oh I guess not. I just didn't touch it, b/c it's the same as pyamqp.

Reply:
updated